### PR TITLE
Add how to use tensorboard locally with sshfs

### PR DESCRIPTION
**Pull request by mdiazmel**
Add some paragraphs about how to use Tensorboard locally, after mounting a remote folder with sshfs.

### CODE REVIEW

File: docs/tips-and-tricks.md [hunk 1]
```diff
@@ -268,12 +268,42 @@ modification ou suppression de machines".
...
+```
+
+3. Launch tensorboard locally, in the host machine. 
+```
+tensorboard --logdir=$HOME/jeanzay/logs
+```
+
+A new browser window will open and show the progress of your training.
+
+#### Use tensorbiad dev
```
Comment (remicres) [inline docs/tips-and-tricks.md:305]: "Use tensorboard dev"

File: docs/tips-and-tricks.md [hunk 2]
```diff
@@ -268,12 +268,42 @@ modification ou suppression de machines".
...
 
-However, if this feels cumbersome, Tensorboard dev might be an alternative.
 [Tensoboard dev](https://tensorboard.dev/) is a tool allowing you to upload your
```
Comment (remicres) [inline docs/tips-and-tricks.md:307]: "[Tensorboard dev](...)"

File: docs/tips-and-tricks.md [hunk 3]
```diff
@@ -268,12 +268,42 @@ modification ou suppression de machines".
 
 ### Tensoboard dev
```
Comment (remicres) [inline docs/tips-and-tricks.md:269]: "Tensorboard"

**Review by remicres — APPROVED** [review]

File: docs/tips-and-tricks.md (hunk 1 above)
Comment (mdiazmel) [reply docs/tips-and-tricks.md]: Thanks @remicres! 
Done in db891d1

**Comment by mdiazmel** [comment]
I'll merge this PR!

**Comment by lesteve** [comment]
Great thanks a lot!

My personal opinion (and probably most of the users I know) is that Tensorboard within `idrjup` is the least convenient method so maybe one day I'll move it at the bottom of the list.

**Comment by remicres** [comment]
+1

I guess it is great for courses, tutorials, this kind of use. But for everyday I'll definitively stick with sshfs+tensorboard

**Pull request merged**